udata/tests/api/test_auth_api.py

```diff
@@ -109,8 +108,7 @@ def test_invalid_apikey(self):
 
         self.assert401(response)
         self.assertEqual(response.content_type, 'application/json')
-        self.assertEqual(response.json,
-                         {'status': 401, 'message': 'Invalid API Key'})
+        self.assertIn('message', response.json)
```

Comment:
Yes, standards error messages are handled upstream by Flask-RESTPlus, it's not relevant to test them here and they are subject to change on each upgrade